config/eslintrc_core.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     'rules': {
         // Possible Errors
 
+        'for-direction': 0, // We don't think this is "oossible error".
```

Comment:
Did you mean "possible error"?

I just think it isn't a job of the lint tool, so I agree with your conclusion.